src/main/java/com/learning/configuration/PropertiesReader.java

```diff
@@ -0,0 +1,35 @@
+package com.learning.configuration;
+
+import com.google.inject.Singleton;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Singleton
+public class PropertiesReader {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(PropertiesReader.class);
+  private final String pathToFile;
+
+  public PropertiesReader() {
+    pathToFile = "src/main/resources/application-properties.yaml";
```

Comment:
If you'd be using spring then by convention file should be called application.yaml.
Since you don't I'd rename it to testProperties.yaml